cluster/gce/gci/configure.sh

```diff
@@ -66,8 +66,12 @@ function download-kube-env {
     # Convert the yaml format file into a shell-style file.
     eval $(${PYTHON} -c '''
 import pipes,sys,yaml
-for k,v in yaml.load(sys.stdin).iteritems():
-  print("readonly {var}={value}".format(var = k, value = pipes.quote(str(v))))
+if sys.version_info[0] < 3:
```

Comment:
Can we add a quick comment explaining that we are checking the python version and invoking the appropriate versions specific code?

Reply:
Doing it now